<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>Function reference â€¢ acidgenerics</title>


<!-- jquery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
<!-- Bootstrap -->

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous" />

<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script>

<!-- bootstrap-toc -->
<link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script>

<!-- Font Awesome icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous" />

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script>

<!-- headroom.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script>



  <link href="../extra.css" rel="stylesheet">
  

<meta property="og:title" content="Function reference" />




<!-- mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->



  </head>

  <body data-spy="scroll" data-target="#toc">
    <div class="container template-reference-index">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">acidgenerics</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.3.7</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../index.html">
    <span class="fas fa fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/acidgenomics/acidgenerics/">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
      
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header>

<div class="row">
  <div class="contents col-md-9">
    <div class="page-header">
      <h1>Reference</h1>
    </div>

    <table class="ref-index">

    <colgroup>
      
      <col class="alias" />
      <col class="title" />
    </colgroup>

    <tbody>
      <tr>
        <th colspan="2">
          <h2 id="section-all-functions" class="hasAnchor"><a href="#section-all-functions" class="anchor"></a>All functions</h2>
          <p class="section-desc"></p>
        </th>
      </tr>
      
      
    </tbody><tbody>
      
      
      <tr>
        
        <td>
          <p><code><a href="aggregateCellsToSamples.html">aggregateCellsToSamples()</a></code> </p>
        </td>
        <td><p>Aggregate cells to samples</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="aggregateCols.html">aggregateCols()</a></code> </p>
        </td>
        <td><p>Aggregate columns</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="aggregateRows.html">aggregateRows()</a></code> </p>
        </td>
        <td><p>Aggregate rows</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="alphaSummary.html">alphaSummary()</a></code> </p>
        </td>
        <td><p>Alpha level cutoff summary statistics</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="alphaThreshold.html">alphaThreshold()</a></code> <code><a href="alphaThreshold.html">`alphaThreshold&lt;-`()</a></code> </p>
        </td>
        <td><p>Alpha threshold</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="as.DataFrame.html">as.DataFrame()</a></code> </p>
        </td>
        <td><p>Coerce object to S4 DataFrame</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="as.SummarizedExperiment.html">as.SummarizedExperiment()</a></code> </p>
        </td>
        <td><p>Coerce object to SummarizedExperiment</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="atomize.html">atomize()</a></code> </p>
        </td>
        <td><p>Atomize</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="autopadZeros.html">autopadZeros()</a></code> </p>
        </td>
        <td><p>Autopad zeros</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="barcodeRanksPerSample.html">barcodeRanksPerSample()</a></code> </p>
        </td>
        <td><p>Barcode ranks per sample</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="calculateMetrics.html">calculateMetrics()</a></code> </p>
        </td>
        <td><p>Calculate quality control metrics</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="camelCase.html">camelCase()</a></code> </p>
        </td>
        <td><p>Camel case</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="capitalize.html">capitalize()</a></code> </p>
        </td>
        <td><p>Capitalize</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="cell2sample.html">cell2sample()</a></code> </p>
        </td>
        <td><p>Cell-to-sample mappings</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="cellCountsPerCluster.html">cellCountsPerCluster()</a></code> </p>
        </td>
        <td><p>Cell counts per cluster</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="cellTypesPerCluster.html">cellTypesPerCluster()</a></code> </p>
        </td>
        <td><p>Cell types per cluster</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="clusters.html">clusters()</a></code> </p>
        </td>
        <td><p>Cluster identifiers</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="coerceToList.html">coerceToList()</a></code> </p>
        </td>
        <td><p>Coerce to list</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="collapseToString.html">collapseToString()</a></code> </p>
        </td>
        <td><p>Collapse to string</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="collectionNames.html">collectionNames()</a></code> <code><a href="collectionNames.html">`collectionNames&lt;-`()</a></code> </p>
        </td>
        <td><p>Gene set collection names</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="contrastName.html">contrastName()</a></code> <code><a href="contrastName.html">`contrastName&lt;-`()</a></code> </p>
        </td>
        <td><p>Contrast name</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="contrastNames.html">contrastNames()</a></code> <code><a href="contrastNames.html">`contrastNames&lt;-`()</a></code> </p>
        </td>
        <td><p>Contrast names</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="contrastSamples.html">contrastSamples()</a></code> </p>
        </td>
        <td><p>Samples corresponding to a differential expression contrast</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="convertGenesToSymbols.html">convertGenesToSymbols()</a></code> </p>
        </td>
        <td><p>Convert genes to symbols</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="convertSampleIDsToNames.html">convertSampleIDsToNames()</a></code> </p>
        </td>
        <td><p>Convert sample identifiers to names</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="convertSymbolsToGenes.html">convertSymbolsToGenes()</a></code> </p>
        </td>
        <td><p>Convert symbols to genes</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="convertTranscriptsToGenes.html">convertTranscriptsToGenes()</a></code> </p>
        </td>
        <td><p>Convert transcripts to genes</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="correlation.html">correlation()</a></code> </p>
        </td>
        <td><p>Correlation</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="cpm.html">cpm()</a></code> </p>
        </td>
        <td><p>Counts per million</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="deg.html">deg()</a></code> </p>
        </td>
        <td><p>Differentially expressed genes</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="diffExp.html">diffExp()</a></code> </p>
        </td>
        <td><p>Differential expression</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="diffExpPerCluster.html">diffExpPerCluster()</a></code> </p>
        </td>
        <td><p>Differential expression per cluster</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="dottedCase.html">dottedCase()</a></code> </p>
        </td>
        <td><p>Dotted case</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="encode.html">encode()</a></code> </p>
        </td>
        <td><p>Encode using run-length encoding</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="enrichedGeneSets.html">enrichedGeneSets()</a></code> </p>
        </td>
        <td><p>Enriched gene sets</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="export.html">export()</a></code> </p>
        </td>
        <td><p>Export</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="factorize.html">factorize()</a></code> </p>
        </td>
        <td><p>Factorize</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="filterCells.html">filterCells()</a></code> </p>
        </td>
        <td><p>Filter cells</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="findMarkers.html">findMarkers()</a></code> </p>
        </td>
        <td><p>Find cluster-specific marker genes</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="foldChangeToLogRatio.html">foldChangeToLogRatio()</a></code> </p>
        </td>
        <td><p>Fold change to log ratio</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="fpkm.html">fpkm()</a></code> </p>
        </td>
        <td><p>Fragments per kilobase per million mapped fragments</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="geneNames.html">geneNames()</a></code> </p>
        </td>
        <td><p>Gene names</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="geometricMean.html">geometricMean()</a></code> </p>
        </td>
        <td><p>Geometric mean</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="headtail.html">headtail()</a></code> </p>
        </td>
        <td><p>Return the first and last parts of an object</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="humanize.html">humanize()</a></code> </p>
        </td>
        <td><p>Humanize an R object</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="interestingGroups.html">interestingGroups()</a></code> <code><a href="interestingGroups.html">`interestingGroups&lt;-`()</a></code> </p>
        </td>
        <td><p>Interesting groups</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="join.html">innerJoin()</a></code> <code><a href="join.html">leftJoin()</a></code> <code><a href="join.html">rightJoin()</a></code> <code><a href="join.html">fullJoin()</a></code> <code><a href="join.html">semiJoin()</a></code> <code><a href="join.html">antiJoin()</a></code> </p>
        </td>
        <td><p>Join operations</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="kebabCase.html">kebabCase()</a></code> </p>
        </td>
        <td><p>Kebab case</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="lfcThreshold.html">lfcThreshold()</a></code> <code><a href="lfcThreshold.html">`lfcThreshold&lt;-`()</a></code> </p>
        </td>
        <td><p>Log2 fold change threshold</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="logRatioToFoldChange.html">logRatioToFoldChange()</a></code> </p>
        </td>
        <td><p>Log ratio to fold change</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="makeDimnames.html">makeDimnames()</a></code> </p>
        </td>
        <td><p>Make syntactically valid dimnames</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="makeLabel.html">makeLabel()</a></code> </p>
        </td>
        <td><p>Make a plot axis or legend label out of a character string</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="makeNames.html">makeNames()</a></code> </p>
        </td>
        <td><p>Make syntactically valid names</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="makeTitle.html">makeTitle()</a></code> </p>
        </td>
        <td><p>Make a title out of character string</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="makeWords.html">makeWords()</a></code> </p>
        </td>
        <td><p>Convert syntactic names to words separated by spaces</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="mapGenesToIDs.html">mapGenesToIDs()</a></code> </p>
        </td>
        <td><p>Map genes (gene names) to gene identifiers</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="mapGenesToRownames.html">mapGenesToRownames()</a></code> </p>
        </td>
        <td><p>Map genes to row names</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="mapGenesToSymbols.html">mapGenesToSymbols()</a></code> </p>
        </td>
        <td><p>Map genes (gene identifiers) to gene names (symbols)</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="markdown.html">markdown()</a></code> </p>
        </td>
        <td><p>Markdown</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="matchRowNameColumn.html">matchRowNameColumn()</a></code> </p>
        </td>
        <td><p>Match row name column</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="mcolnames.html">mcolnames()</a></code> <code><a href="mcolnames.html">`mcolnames&lt;-`()</a></code> </p>
        </td>
        <td><p>Metadata column names</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="melt.html">melt()</a></code> </p>
        </td>
        <td><p>Melt columns into key-value pairs</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="metadata2.html">metadata2()</a></code> <code><a href="metadata2.html">`metadata2&lt;-`()</a></code> </p>
        </td>
        <td><p>Metadata</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="metrics.html">metrics()</a></code> </p>
        </td>
        <td><p>Quality control metrics</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="metricsPerSample.html">metricsPerSample()</a></code> </p>
        </td>
        <td><p>Quality control metrics per sample</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="mutate.html">mutateAll()</a></code> <code><a href="mutate.html">mutateAt()</a></code> <code><a href="mutate.html">mutateIf()</a></code> <code><a href="mutate.html">transmuteAt()</a></code> <code><a href="mutate.html">transmuteIf()</a></code> </p>
        </td>
        <td><p>Mutate multiple columns</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="nonzeroRowsAndCols.html">nonzeroRowsAndCols()</a></code> </p>
        </td>
        <td><p>Subset object to keep only non-zero rows and columns</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plot5Prime3PrimeBias.html">plot5Prime3PrimeBias()</a></code> </p>
        </td>
        <td><p>Plot 5' to 3' bias</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotBarcodeRanks.html">plotBarcodeRanks()</a></code> </p>
        </td>
        <td><p>Plot barcode ranks</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotCellCounts.html">plotCellCounts()</a></code> </p>
        </td>
        <td><p>Plot cell counts</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotCellCountsPerCluster.html">plotCellCountsPerCluster()</a></code> </p>
        </td>
        <td><p>Plot cell counts per cluster</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotCellTypesPerCluster.html">plotCellTypesPerCluster()</a></code> </p>
        </td>
        <td><p>Plot cell types per cluster</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotCells.html">plotCells()</a></code> </p>
        </td>
        <td><p>Plot cells</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotCorrelationHeatmap.html">plotCorrelationHeatmap()</a></code> </p>
        </td>
        <td><p>Correlation heatmap</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotCountDensity.html">plotCountDensity()</a></code> </p>
        </td>
        <td><p>Plot count density</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotCounts.html">plotCounts()</a></code> </p>
        </td>
        <td><p>Plot counts</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotCountsCorrelation.html">plotCountsCorrelation()</a></code> </p>
        </td>
        <td><p>Plot counts correlation</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotCountsCorrelationHeatmap.html">plotCountsCorrelationHeatmap()</a></code> </p>
        </td>
        <td><p>Plot counts correlation heatmap</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotCountsPerBiotype.html">plotCountsPerBiotype()</a></code> </p>
        </td>
        <td><p>Plot counts per biotype</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotCountsPerBroadClass.html">plotCountsPerBroadClass()</a></code> </p>
        </td>
        <td><p>Plot counts per broad class definition</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotCountsPerCell.html">plotCountsPerCell()</a></code> </p>
        </td>
        <td><p>Plot counts per cell</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotCountsPerFeature.html">plotCountsPerFeature()</a></code> </p>
        </td>
        <td><p>Plot counts per feature</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotCountsPerGene.html">plotCountsPerGene()</a></code> </p>
        </td>
        <td><p>Plot counts per gene</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotCountsVsFeatures.html">plotCountsVsFeatures()</a></code> </p>
        </td>
        <td><p>Plot count and feature correlation</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotDEGHeatmap.html">plotDEGHeatmap()</a></code> </p>
        </td>
        <td><p>Differentially expressed gene heatmap</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotDEGPCA.html">plotDEGPCA()</a></code> </p>
        </td>
        <td><p>Plot differentially expressed gene principal component analysis</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotDEGUpset.html">plotDEGUpset()</a></code> </p>
        </td>
        <td><p>UpSet plot of directional DEG intersections across contrasts</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotDots.html">plotDots()</a></code> </p>
        </td>
        <td><p>Dot plot</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotEnrichedGeneSets.html">plotEnrichedGeneSets()</a></code> </p>
        </td>
        <td><p>Plot enriched gene sets</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotEnrichedUpset.html">plotEnrichedUpset()</a></code> </p>
        </td>
        <td><p>UpSet plot of directional enriched pathway intersections across contrasts</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotExonicMappingRate.html">plotExonicMappingRate()</a></code> </p>
        </td>
        <td><p>Plot exonic mapping rate</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotFeature.html">plotFeature()</a></code> </p>
        </td>
        <td><p>Plot feature</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotFeaturesDetected.html">plotFeaturesDetected()</a></code> </p>
        </td>
        <td><p>Plot features detected</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotFeaturesPerCell.html">plotFeaturesPerCell()</a></code> </p>
        </td>
        <td><p>Plot features per cell</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotGenderMarkers.html">plotGenderMarkers()</a></code> </p>
        </td>
        <td><p>Plot sexually dimorphic gender marker genes</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotGeneSaturation.html">plotGeneSaturation()</a></code> </p>
        </td>
        <td><p>Plot gene detection saturation</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotHeatmap.html">plotHeatmap()</a></code> </p>
        </td>
        <td><p>Heatmap</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotIntergenicMappingRate.html">plotIntergenicMappingRate()</a></code> </p>
        </td>
        <td><p>Plot intergenic mapping rate</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotIntronicMappingRate.html">plotIntronicMappingRate()</a></code> </p>
        </td>
        <td><p>Plot intronic mapping rate</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotKnownMarkers.html">plotKnownMarkers()</a></code> </p>
        </td>
        <td><p>Plot known markers</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotMappedReads.html">plotMappedReads()</a></code> </p>
        </td>
        <td><p>Plot mapped reads</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotMappingRate.html">plotMappingRate()</a></code> </p>
        </td>
        <td><p>Plot mapping rate</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotMarker.html">plotMarker()</a></code> </p>
        </td>
        <td><p>Plot cell-type-specific gene markers</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotMeanSD.html">plotMeanSD()</a></code> </p>
        </td>
        <td><p>Plot row standard deviations vs. row means</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotMitoRatio.html">plotMitoRatio()</a></code> </p>
        </td>
        <td><p>Plot mitochondrial transcript abundance</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotMitoVsCoding.html">plotMitoVsCoding()</a></code> </p>
        </td>
        <td><p>Plot mitochondrial vs. coding counts</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotNovelty.html">plotNovelty()</a></code> </p>
        </td>
        <td><p>Plot novelty score</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotPCACovariates.html">plotPCACovariates()</a></code> </p>
        </td>
        <td><p>Find correlation between principal components (PCs) and covariates</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotPCElbow.html">plotPCElbow()</a></code> </p>
        </td>
        <td><p>Plot principal component elbow</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotQC.html">plotQC()</a></code> </p>
        </td>
        <td><p>Quality control</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotQuantileHeatmap.html">plotQuantileHeatmap()</a></code> </p>
        </td>
        <td><p>Quantile heatmap</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotRRNAMappingRate.html">plotRRNAMappingRate()</a></code> </p>
        </td>
        <td><p>Plot ribosomal RNA (rRNA) mapping rate</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotReadsPerCell.html">plotReadsPerCell()</a></code> </p>
        </td>
        <td><p>Plot read counts per cell</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotReducedDim.html">plotReducedDim()</a></code> </p>
        </td>
        <td><p>Plot reduced dimensions</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotStackedBarPlot.html">plotStackedBarPlot()</a></code> </p>
        </td>
        <td><p>Plot stacked bar plot</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotSums.html">plotSums()</a></code> </p>
        </td>
        <td><p>Plot row or column sums</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotTSNE.html">plotTSNE()</a></code> </p>
        </td>
        <td><p>t-SNE plot</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotTopMarkers.html">plotTopMarkers()</a></code> </p>
        </td>
        <td><p>Plot top markers</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotTotalCounts.html">plotTotalCounts()</a></code> </p>
        </td>
        <td><p>Plot total read counts</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotTotalReads.html">plotTotalReads()</a></code> </p>
        </td>
        <td><p>Plot total reads</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotUMAP.html">plotUMAP()</a></code> </p>
        </td>
        <td><p>UMAP plot</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotUpset.html">plotUpset()</a></code> </p>
        </td>
        <td><p>UpSet plot</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotViolin.html">plotViolin()</a></code> </p>
        </td>
        <td><p>Violin plot</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotVolcano.html">plotVolcano()</a></code> </p>
        </td>
        <td><p>Volcano plot</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotWaterfall.html">plotWaterfall()</a></code> </p>
        </td>
        <td><p>Plot waterfall</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plotZerosVsDepth.html">plotZerosVsDepth()</a></code> </p>
        </td>
        <td><p>Plot percentage of zeros vs. library depth</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="relativeLogExpression.html">relativeLogExpression()</a></code> </p>
        </td>
        <td><p>Relative log expression</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="removeNA.html">removeNA()</a></code> </p>
        </td>
        <td><p>Remove rows and columns containing only NA values</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="results.html">results()</a></code> </p>
        </td>
        <td><p>Results</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="resultsMatrix.html">resultsMatrix()</a></code> </p>
        </td>
        <td><p>Results matrix</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="resultsNames.html">resultsNames()</a></code> <code><a href="resultsNames.html">`resultsNames&lt;-`()</a></code> </p>
        </td>
        <td><p>Results names</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="resultsTables.html">resultsTables()</a></code> </p>
        </td>
        <td><p>Results tables</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="sampleData.html">sampleData()</a></code> <code><a href="sampleData.html">`sampleData&lt;-`()</a></code> </p>
        </td>
        <td><p>Sample data</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="sanitizeNA.html">sanitizeNA()</a></code> </p>
        </td>
        <td><p>Sanitize NA values</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="sanitizePercent.html">sanitizePercent()</a></code> </p>
        </td>
        <td><p>Sanitize percentage</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="select.html">selectIf()</a></code> </p>
        </td>
        <td><p>Select multiple columns</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="selectSamples.html">selectSamples()</a></code> </p>
        </td>
        <td><p>Select samples</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="sentenceCase.html">sentenceCase()</a></code> </p>
        </td>
        <td><p>Sentence case</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="snakeCase.html">snakeCase()</a></code> </p>
        </td>
        <td><p>Snake case</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="stripTranscriptVersions.html">stripTranscriptVersions()</a></code> </p>
        </td>
        <td><p>Strip transcript versions</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="subsetPerSample.html">subsetPerSample()</a></code> </p>
        </td>
        <td><p>Subset per sample</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="tmm.html">tmm()</a></code> </p>
        </td>
        <td><p>Trimmed mean of M-values</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="topCellsPerSample.html">topCellsPerSample()</a></code> </p>
        </td>
        <td><p>Top cells per sample</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="topMarkers.html">topMarkers()</a></code> </p>
        </td>
        <td><p>Top markers</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="topTables.html">topTables()</a></code> </p>
        </td>
        <td><p>Top tables</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="tpm.html">tpm()</a></code> </p>
        </td>
        <td><p>Transcripts per million</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="uniteInterestingGroups.html">uniteInterestingGroups()</a></code> </p>
        </td>
        <td><p>Unite interesting groups into a single column</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="upperCamelCase.html">upperCamelCase()</a></code> </p>
        </td>
        <td><p>Upper camel case</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="zerosVsDepth.html">zerosVsDepth()</a></code> </p>
        </td>
        <td><p>Percentage of zeros vs. library depth</p></td>
      </tr>
    </tbody>
    </table>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top">
      <h2 data-toc-skip>Contents</h2>
    </nav>
  </div>
</div>


      <footer>
      <div class="copyright">
  <p>Developed by <a href='https://steinbaugh.com/'>Michael Steinbaugh</a>, <a href='https://acidgenomics.com/'>Acid Genomics</a>.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.5.1.</p>
</div>

      </footer>
   </div>

  


  </body>
</html>


